docker-compose/adaptive-sampling/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+BINARY ?= all-in-one
```

Comment:
all-in-one is a v1 style binary. I would prefer we test v2 version (or at least both, but v2 is higher priority and testing v1 at this point is wasted work since v1 will be EOLed in a year)

Reply:
@yurishkuro I was trying to test `jaeger` binary but I can't seem to access port 14268. Do you know why that is? I left more details on the issue: https://github.com/jaegertracing/jaeger/issues/5717.